core/templates/components/state-directives/response-header/response-header.component.html

```diff
@@ -22,7 +22,7 @@
   </div>
 
   <br class="break-in-mobile">
-  <span *ngIf="outcome"> →</span>
+  <span *ngIf="outcome">→&nbsp;</span>
```

Comment:
Would a regular space work here?

Reply:
@seanlip A regular space character is not working here. That's why i have used a this.

Reply:
Then can we use CSS to add some padding instead of using nbsp?